firefox-ios/Client/Frontend/Settings/ContentBlockerSettingViewController.swift

```diff
@@ -5,9 +5,15 @@
 import Common
 import Foundation
 import Shared
+import ComponentLibrary
 
-class ContentBlockerSettingViewController: SettingsTableViewController {
-    private let button = UIButton()
+class ContentBlockerSettingViewController: SettingsTableViewController, ThemeApplicable {
```

Comment:
since `SettingsTableViewController` is a `ThemedTableViewController`, we can remove `ThemeApplicable` and can override the `applyTheme` from the parent class.